CoreScriptsRoot/Modules/ChatSelector.lua

```diff
@@ -0,0 +1,203 @@
+local USE_NEW_CHAT = false --and settings():GetFFlag("UseNewChat")
```

Comment:
It was commented out because the flag doesn't exist yet. It's just there right now as a reminder to implement later. I guess a comment there also would have been good. I'll be sure to update it to that format later when the flag is created.